tests/phpunit/tests/blocks/render.php

```diff
@@ -218,9 +218,9 @@ public function test_do_block_output( $html_filename, $server_html_filename ) {
 
 		$html = do_blocks( self::strip_r( file_get_contents( $html_path ) ) );
 		// If blocks opt into Gutenberg's layout implementation
-		// the container will receive an added classname of `wp_unique_id( 'wp-container-' )`
+		// the container will receive an added classname of `wp_unique_id( 'wp-container-[blockname]-layout' )`
```

Comment:
```suggestion
		// the container will receive an added classname of `wp_unique_id( 'wp-container-[blockname]-layout' )`
```
This function name is no longer relevant here. To save reference to implementation, I'd suggest this instead:

```suggestion
		// the container will receive an additional, unique classname based on "wp-container-[blockname]-layout"
```

Additionally, the next line mentions an `id`, but it's actually referring to a classname, so this could be changed for accuracy while we're here.

Reply:
Partially fixed in [2d3db44](https://github.com/WordPress/wordpress-develop/pull/5528/commits/2d3db4418b82b8860fd10109cdb9fefaa4b2be09). The `id` on the next line wasn't changed, but this isn't a critical issue.